ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/response/JsonRpcError.java

```diff
@@ -162,6 +162,8 @@ public enum JsonRpcError {
   VALUE_NOT_ZERO(-50100, "We cannot transfer ether in a private transaction yet."),
 
   CANT_CONNECT_TO_LOCAL_PEER(-32100, "Cannot add local node as peer."),
+  CANT_RESOLVE_PEER_ENODE_DNS(-32100, "Cannot resolve enode DNS hostname"),
+  CANT_USE_PEER_ENODE_DNS(-32100, "Enode DNS support is disabled"),
```

Comment:
```suggestion
  DNS_NOT_ENABLED(-32100, "Enode DNS support is disabled"),
```

Reply:
suggestion for a simpler name. also "X not enabled" matches other error codes

Reply:
Thanks, new commit renames the constant and updates the references to it.